curriculum/challenges/english/01-responsive-web-design/css-flexbox/align-elements-using-the-justify-content-property.md

```diff
@@ -11,7 +11,7 @@ dashedName: align-elements-using-the-justify-content-property
 
 Sometimes the flex items within a flex container do not fill all the space in the container. It is common to want to tell CSS how to align and space out the flex items a certain way. Fortunately, the `justify-content` property has several options to do this. But first, there is some important terminology to understand before reviewing those options.
 
-[For more info about flex-box properties](https://www.freecodecamp.org/news/flexbox-the-ultimate-css-flex-cheatsheet/)
+<a href="https://www.freecodecamp.org/news/flexbox-the-ultimate-css-flex-cheatsheet/" target="_blank" rel="noopener noreferrer nofollow">For more info about flex-box properties</a>
```

Comment:
The sentence looks uncomplete

For more info about flex-box properties read here?
```suggestion
<a href="https://www.freecodecamp.org/news/flexbox-the-ultimate-css-flex-cheatsheet/" target="_blank" rel="noopener noreferrer nofollow">For more info about flex-box properties</a> read here.
```